scripts/perftests/PerformanceTestUtilities.ps1

```diff
@@ -479,9 +484,49 @@ Function RunRestore(
         $staticGraphOutputValue = "N/A"
     }
 
+    If($isClientDotnetExe -Or $isClientMSBuild)
+    {
+        # Always disable NuGetAudit. It's something impacted by the time of execution, and not helpful in these type of performance tests.
+        $arguments.Add("/p:NuGetAudit=false")   
+    }
+
+    if(($isClientDotnetExe -Or $isClientMSBuild) -And $forceLegacyResolverFallback)
+    {
+        $arguments.Add("/p:RestoreUseLegacyDependencyResolver=true")
+    }
+
+    if($useLocallyBuiltNuGet)
+    {
+        if($isClientMSBuild)
+        {
+            $NETFramework = "net472"
+            $Configuration = "release"
+            $packDllPath = Join-Path $NuGetClientRoot "artifacts\NuGet.Build.Tasks.Pack\bin\$Configuration\$NETFramework\NuGet.Build.Tasks.Pack.dll"
+            $packTargetsPath = Join-Path $NuGetClientRoot "src\NuGet.Core\NuGet.Build.Tasks.Pack\NuGet.Build.Tasks.Pack.targets"
+            $restoreDllPath = Join-Path $NuGetClientRoot "artifacts\NuGet.Build.Tasks\bin\$Configuration\$NETFramework\NuGet.Build.Tasks.dll"
+            $nugetRestoreTargetsPath = Join-Path $NuGetClientRoot "src\NuGet.Core\NuGet.Build.Tasks\NuGet.targets"
+            $nugetPropsPath = Join-Path $NuGetClientRoot "src\NuGet.Core\NuGet.Build.Tasks\NuGet.props"
+            $consoleExePath = Join-Path $NuGetClientRoot "artifacts\NuGet.Build.Tasks.Console\bin\$Configuration\$NETFramework\NuGet.Build.Tasks.Console.exe"
+            $arguments.Add("/p:NuGetRestoreTargets=$nugetRestoreTargetsPath");
+            $arguments.Add("/p:NuGetPropsFile=$nugetPropsPath");
+            $arguments.Add("/p:RestoreTaskAssemblyFile=$restoreDllPath");
```

Comment:
`RestoreTaskAssembly` is set by `NuGet.targets`, so it's unnecessary to explicitly set it here

Reply:
I think the targets I'm pointing too are from the src folder rather than the output. 

I can try to fix this.